hw_diag/templates/diagnostics_page.html

```diff
@@ -88,7 +88,11 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                 </tr>
                 <tr>
                   <td>Frequency</td>
-                  <td class="text-right">{{ diagnostics.FR }}</td>
+                  {% if diagnostics.FREQ %}
```

Comment:
shall we completely remove this. We already have "Region Plan" which will have same value.

Reply:
Came to say the same. We should remove it completely. Otherwise, looks good to me.